addons/base/views.py

```diff
@@ -229,50 +242,43 @@ def make_auth(user):
         }
     return {}
 
-def download_is_from_mfr(req, payload):
-    metrics_data = payload['metrics']
-    uri = metrics_data['uri']
-    is_render_uri = furl.furl(uri or '').query.params.get('mode') == 'render'
-    return (
-        # This header is sent for download requests that
-        # originate from MFR, e.g. for the code pygments renderer
-        req.headers.get('X-Cos-Mfr-Render-Request', None) or
-        # Need to check the URI in order to account
-        # for renderers that send XHRs from the
-        # rendered content, e.g. PDFs
-        is_render_uri
-    )
```

Comment:
This function (and the conditional checking it) was previously preventing in-app renders from being recorded in Metrics as a download.